packages/wallet-sdk/src/sign/scw/message/type/Action.ts

```diff
@@ -127,6 +131,36 @@ export type AddEthereumChainAction = {
   };
 };
 
+export type GetCapacitiesAction = {
+  method: SupportedEthereumMethods.WalletGetCapacities;
+  params: null;
+};
+
+export type WalletSendTransactionAction = {
+  method: SupportedEthereumMethods.WalletSendTransaction;
+  params: {
+    chainId: number;
+    sender: string;
+    calls: {
+      target: string;
+      value: string;
+      data: string;
+      gas: string;
```

Comment:
gas is optional and top level (with chainId and sender). and need a version string, top level too.

Reply:
thank you, addressed in [cd1eb13](https://github.com/coinbase/coinbase-wallet-sdk/pull/1138/commits/cd1eb1360fb42b48a38b88c0bc59db3c2c3f9b1f)